src/App.js

```diff
@@ -2,14 +2,21 @@ import "./App.css";
 import { WelcomeUser } from "./pages/WelcomeUser";
 import { UserOnboarding } from "./components/UserOnboarding";
 import { useApp } from "./context/app-context";
+import { useEffect, useState } from "react";
+import { imageDb } from "./db/imageDb";
 
 function App() {
```

Comment:
Just so that App.js looks clean, if required, in future you can make a component for background generator as well rather than writing code in app.js

Reply:
sure will do that